src/compiler/scala/tools/nsc/backend/jvm/BCodeIdiomatic.scala

```diff
@@ -23,7 +23,7 @@ import scala.tools.nsc.backend.icode.Primitives.{NE, EQ, TestOp, ArithmeticOp}
  */
 trait BCodeIdiomatic {
   val int: BackendInterface
-  lazy val bTypes = new BTypesFromSymbols[int.type](int)
+  final lazy val bTypes = new BTypesFromSymbols[int.type](int)
```

Comment:
Hmm, IIRC this was something fundamental to Dotty that we couldn't get around. I ended up asking @odersky who said it should be final.

Compiling without it will give you an error message - if you can compile it without `final`, that'd jog my memory.